### DISCUSSION TITLE
Unable to translate

### DISCUSSION THREAD

**Discussion by zhuoyuejiejie** [SubtitleEdit/subtitleedit#6299] [Q&A]
Hello,In recent days, I have a problem that I cannot translate into other languages during the use，it shows free API quota exceeded,how to solve this problem?

**Comment by niksedk** [comment]
You can wait a while...
Or try some of the translation plugs, or setup api key at Google, Microsoft, or DeepL. 
https://www.nikse.dk/subtitleedit/online might also work.

**Reply by zhuoyuejiejie** [reply]
Thanks
